src/core/server/utils/index.ts

```diff
@@ -1,10 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0 and the Server Side Public License, v 1; you may not use this file except
- * in compliance with, at your election, the Elastic License 2.0 or the Server
- * Side Public License, v 1.
- */
-
-export * from './from_root';
-export * from './package_json';
```

Comment:
should be removed from https://github.com/elastic/kibana/blob/510caae1e06274be173760f854795ae8583fa65b/.eslintrc.js#L427-L428